tools/StaticAnalysis/Exceptions/Az.Network/BreakingChangeIssues.csv

```diff
@@ -4,4 +4,6 @@
 "Az.Network","Microsoft.Azure.Commands.Network.NewAzureRmLoadBalancerRuleConfigCommand","New-AzLoadBalancerRuleConfig","0","2020","The cmdlet 'New-AzLoadBalancerRuleConfig' no longer supports the type 'System.String' for parameter 'BackendAddressPoolId'.","Change the type for parameter 'BackendAddressPoolId' back to 'System.String'."
 "Az.Network","Microsoft.Azure.Commands.Network.NewAzureRmLoadBalancerRuleConfigCommand","New-AzLoadBalancerRuleConfig","0","2020","The cmdlet 'New-AzLoadBalancerRuleConfig' no longer supports the type 'Microsoft.Azure.Commands.Network.Models.PSBackendAddressPool' for parameter 'BackendAddressPool'.","Change the type for parameter 'BackendAddressPool' back to 'Microsoft.Azure.Commands.Network.Models.PSBackendAddressPool'."
 "Az.Network","Microsoft.Azure.Commands.Network.SetAzureRmLoadBalancerRuleConfigCommand","Set-AzLoadBalancerRuleConfig","0","2020","The cmdlet 'Set-AzLoadBalancerRuleConfig' no longer supports the type 'System.String' for parameter 'BackendAddressPoolId'.","Change the type for parameter 'BackendAddressPoolId' back to 'System.String'."
-"Az.Network","Microsoft.Azure.Commands.Network.SetAzureRmLoadBalancerRuleConfigCommand","Set-AzLoadBalancerRuleConfig","0","2020","The cmdlet 'Set-AzLoadBalancerRuleConfig' no longer supports the type 'Microsoft.Azure.Commands.Network.Models.PSBackendAddressPool' for parameter 'BackendAddressPool'.","Change the type for parameter 'BackendAddressPool' back to 'Microsoft.Azure.Commands.Network.Models.PSBackendAddressPool'."
\ No newline at end of file
+"Az.Network","Microsoft.Azure.Commands.Network.SetAzureRmLoadBalancerRuleConfigCommand","Set-AzLoadBalancerRuleConfig","0","2020","The cmdlet 'Set-AzLoadBalancerRuleConfig' no longer supports the type 'Microsoft.Azure.Commands.Network.Models.PSBackendAddressPool' for parameter 'BackendAddressPool'.","Change the type for parameter 'BackendAddressPool' back to 'Microsoft.Azure.Commands.Network.Models.PSBackendAddressPool'."
+"Az.Network","Microsoft.Azure.Commands.Network.NewAzureLoadBalancerBackendAddress","New-AzLoadBalancerBackendAddressConfig","0","1060","The parameter set 'SetByResourcePublicIpAddress' for cmdlet 'New-AzLoadBalancerBackendAddressConfig' is no longer the default parameter set.","Change the default parameter for cmdlet 'New-AzLoadBalancerBackendAddressConfig' back to 'SetByResourcePublicIpAddress'."
+"Az.Network","Microsoft.Azure.Commands.Network.RemoveAzureLoadBalancerBackendPool","Remove-AzLoadBalancerBackendAddressPool","0","1050","The parameter set 'DeleteByInputObjectParameterSet' for cmdlet 'Remove-AzLoadBalancerBackendAddressPool' has been removed.","Add parameter set 'DeleteByInputObjectParameterSet' back to cmdlet 'Remove-AzLoadBalancerBackendAddressPool'."
```

Comment:
Changing the default parameter set is not a breaking change, neither is making `-InputObject` mandatory in its parameter set, because otherwise the cmdlet would error out.